### ISSUE TITLE
Mermaid: stateDiagram is not supported

### ISSUE THREAD

**Issue by AlexCzar** [asciidoctor/asciidoctor-diagram#272]
When I try to render mermaid stateDiagram asciidoctor process just hangs.
AsciidoctorJ 2.2.0 (Asciidoctor 2.0.10)
Looks like it is related to mmdc problem.
Kroki renders stateDiagrams fine, so I have to manually pre-render them and then include as images into adoc. Effectively mermaid integration is broken.

**Comment by pepijnve** [comment]
Could you provide an example that's not working?

**Comment by pepijnve** [comment]
This is the same issue as #257. No fix yet I'm afraid.

**Comment by pepijnve** [comment]
FYI, I've made a fix for #257. After that change I get the following syntax error when generating one of the example state diagrams from the mermaid website. I'm not sure what's wrong here.

```
Error: Evaluation failed: Error: Parse error on line 1:
stateDiagram    [*]
^
Expecting 'NEWLINE', 'SPACE', 'GRAPH', got 'ALPHA'
```

**Comment by pepijnve** [comment]
Looks like I'm hitting https://github.com/mermaid-js/mermaid/issues/1235
